### DISCUSSION TITLE
Release 10.7.3

### DISCUSSION THREAD

**Discussion by joshuaboniface** [jellyfin/jellyfin#5967] [General]
# Jellyfin 10.7.3

Stable hotfix release for 10.7.z release branch.

## New Features and Major Improvements

N/A

## Release Notes

* [All] Small bugfixes
* [All] Security advisory https://github.com/jellyfin/jellyfin/security/advisories/GHSA-rgjw-4fwc-9v96: Remove dangerous endpoints that allow unauthenticated enumeration and access to private HTTP resources.  **NOTE**: This is an API-breaking change. The major security risk outweights the issue of changing the endpoints. These were used only when fetching remote images.

## Changelog

GitHub Project: https://github.com/orgs/jellyfin/projects/27

### [jellyfin](https://github.com/jellyfin/jellyfin) [18]

 * [SECURITY GHSA-rgjw-4fwc-9v96] [@joshuaboniface] Remove /Images/Remote API endpoint
 * #5943 [@Maxr1998] Nullability handling for device profile classes
 * #5937 [@Maxr1998] Remove extraneous 'stream' parameter
 * #5904 [@cvium] add UpdatePeopleAsync and add people to both tables
 * #5903 [@iwalton3] Leave SyncPlay group on session disconnect.
 * #5881 [@cvium] Add tvrage and imdb ids for episodes
 * #5878 [@Artiume] Fix Audiobook Resume
 * #5873 [@cvium] Check for empty string when migrating displaypreferences
 * #5870 [@cvium] Fix TMDb Person Provider
 * #5863 [@cvium] use IF NOT EXISTS in migration
 * #5861 [@BaronGreenback] Change profile matching to match what the web interface says.
 * #5860 [@cvium] Fix notification disabled users list
 * #5859 [@cvium] Respect user settings for transcode and remux
 * #5852 [@cvium] Add Person to TypedBaseItems if it's new
 * #5848 [@sgmoore] Fix ArgumentOutOfRangeException scanning AudioBooks
 * #5826 [@BaronGreenback] PlayTo Fix: Use external ip not internal interface
 * #5808 [@cvium] Optimize the way items are grouped into collections

### [jellyfin-web](https://github.com/jellyfin/jellyfin-web) [12]

 * jellyfin/jellyfin-web#2657 [@thornbill] Bump jellyfin-apiclient to 1.8.0
 * jellyfin/jellyfin-web#2648 [@thornbill] Fix alpine python package
 * jellyfin/jellyfin-web#2647 [@thornbill] Fix mov support in Safari
 * jellyfin/jellyfin-web#2645 [@joshuaboniface] Remove image proxying through API
 * jellyfin/jellyfin-web#2624 [@dmitrylyzo] webOS: fix Favorites tab
 * jellyfin/jellyfin-web#2620 [@dmitrylyzo] Hide AlphaPicker on Search page for real TVs
 * jellyfin/jellyfin-web#2619 [@dmitrylyzo] Fix video filters hiding
 * jellyfin/jellyfin-web#2617 [@iwalton3] Set scrollbar width in TV mode.
 * jellyfin/jellyfin-web#2616 [@dmitrylyzo] Fix view style selection dialog (ES6 migration)
 * jellyfin/jellyfin-web#2604 [@Artiume] Update Audiobook Resume Help
 * jellyfin/jellyfin-web#2593 [@oddstr13] Ensure that fillHeight gets set on image requests
 * jellyfin/jellyfin-web#2519 [@dmitrylyzo] Add HEVC in TS for Tizen and webOS

<hr /><em>This discussion was created from the release <a href='https://github.com/jellyfin/jellyfin/releases/tag/v10.7.3'>Release 10.7.3</a>.</em>

**Comment by joshuaboniface** [comment]
This release was broken. Playback was failing. Superceded by https://github.com/jellyfin/jellyfin/releases/tag/v10.7.4

**Reply by itsthejoker** [reply]
Sometimes these things happen. We appreciate you!
